assets/texts/en/privacy.txt

```diff
@@ -1,4 +1,4 @@
-This add-on does not send any information to any third-party or the add-on author.
+This add-on does not send any information to any third-party, or the add-on author.
```

Comment:
Oh the [oxford comma](https://en.wikipedia.org/wiki/Serial_comma)… Well… little American.

I dunno what I even write here, I guess I tend to en_GB, so let's maybe not use this?
```suggestion
This add-on does not send any information to any third-party or the add-on author.
```

Reply:
That's paradoxical... Isn't Oxford from the UK?